packages/app-mobile/components/get-responsive-value.ts

```diff
@@ -0,0 +1,83 @@
+// getResponsiveValue() returns the corresponding value for
```

Comment:
Files should be named like the variable they export, so should be `getResponsiveValue.ts` (the codebase is not consistent for this, but we do that for new code)